keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -339,16 +466,27 @@ def _format_inputs(self, inputs):
             inputs = {IMAGES: inputs}
             return inputs, metadata
 
+        valid_keys = [
+            IMAGES,
+            LABELS,
+            TARGETS,
+            BOUNDING_BOXES,
+            KEYPOINTS,
+            SEGMENTATION_MASKS,
+        ]
+        if not all([x in valid_keys for x in inputs.keys()]):
```

Comment:
I think this is a departure for the current functionality where we just pass-through all the keys we don't know about. What's the value of raising an error here?

Is the idea to alert users of a typo if e.g. they had a key like "buonding_boxes"?

I worry a bit that this is too coercive, but at the same time I can imagine that using a KPL and not having the boxes augmented because of a typo in your dict keys would be very annoying.

Reply:
Reverted